

package dt.spoon.test;

import java.io.IOException;

public class JXTest {
	
	public static boolean next() {
		return true;
	}
	
    public static void entrance() throws Exception {
        long start_time = java.lang.System.nanoTime();
        int x = -5;
        while (x<= 0 && next());
        
        
        for (int i = 0; i< 2; i++) 
        	for (int j=1; j   <   5; j++)
        		for (int f = -1; f < 10; f++) {
        			
        			
        			if (start_time > 1000) return;
        			String t = "xx";
        			try {
        				
        			}
        			catch (Exception e) {
        				throw e;
        			}
        		}
        
        
        for (int w = 1; w < 10000000; w++) {
            int k = 1;
            k = 2;
        }
        try {
        	java.lang.System.out.println((("JX - Completion Time: " 
        + ((((double) ((java.lang.System.nanoTime()) - start_time)) / 1000) / 1000)) + "ms"));
        }
        catch (Exception e) {
        	
        	throw new Exception(e);
        	//TODO
        }
    }

    public static void main(java.lang.String[] args) throws Exception {
        entrance();
        int x = 0;
        try {
        	
        }
        catch (Exception e) {
        	int count = 5;
        	//xx
        	do {   while (count < 10) {}
        		for (int ff = 1; ff<2; ff++) {}
        	} while (false);
        	//xx
        }
    }
    
    class InnerCl {
         InnerCl() {
        	 int a = 0;
         }
    }
}


class Two {
	
}
